opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -87,93 +88,121 @@ message InstrumentationLibraryMetrics {
+    SCALAR_DOUBLE    = 3;
+
+    // HistogramInt64 implies that Metric.Points MUST use
+    // DataPoint.histogram with int64 boundaries.
+    HISTOGRAM_INT64  = 4;
+    // HistogramDouble implies that Metric.Points MUST use
+    // DataPoint.histogram with floating point boundaries.
+    HISTOGRAM_DOUBLE = 5;
+
+    // SummaryInt64 implies that Metric.Points MUST use DataPoint.summary
+    // with int64 min/max/sum/last/count values.
+    SUMMARY_INT64    = 6;
+    // SummaryDouble implies that Metric.Points MUST use
+    // DataPoint.summary with double min/max/sum/last/count values.
+    SUMMARY_DOUBLE   = 7;
+
+    // RawInt64 implies that Metric.Points MUST use DataPoint.raw with
+    // int64 values.
+    RAW_INT64        = 8;
+    // RawDouble implies that Metric.Points MUST use DataPoint.raw with
+    // double value.
+    RAW_DOUBLE       = 9;
   }
 
-  // type is the type of values this metric has.
-  Type type = 4;
+  // ValueType is the type of values this metric has.
+  ValueType value_type = 4;
 
-  // Temporality is the temporal quality values of a metric have. It
-  // describes how those values relate to the time interval over which they
-  // are reported.
-  enum Temporality {
-    // INVALID_TEMPORALITY is the default Temporality, it MUST not be
-    // used.
-    INVALID_TEMPORALITY = 0;
+  // KindElement contains a set of bit masks used to construct Kind enum values.
+  // There are 7 bits used presently, broken into groups:
+  //
+  //   Temporality is the temporal quality values of a metric have. It
+  //   describes how those values relate to the time interval over
+  //   which they are reported.  One of the 3 Temporality values
+  //   (CUMULATIVE, DELTA, INSTANTANEOUS) MUST be set.
+  //
+  //   Structure indicates whether metric data describes a sum of
+  //   measurements (ADDING) or a collection of individual
+  //   measurements (GROUPING).  Value types may be interpreted
+  //   differently depending on Structure.  For example, a Histogram
+  //   DataPoint may be computed for a Counter (ADDING) instrument
+  //   or a ValueRecorder (GROUPING) instrument, and:
+  //
+  //   - ADDING Histograms define a meaningful Sum in the form of a
+  //     distribution of constituent measurements.
+  //   - GROUPING Histograms define a distribution of measurements,
+  //     where the sum is potentially not meaningful.  If also
+  //     SYNCHRONOUS, these data points define a rate in terms of
+  //     their Count.
+  //
+  //   Monotonicity applies only to ADDING data points.  When set,
+  //   MONOTONIC indicates that the calculated sum can be monitored
+  //   as a rate of change.
+  //
+  //   Synchronicity applies to data points produced when the
+  //   application called the API, as opposed to the SDK calling the
+  //   application via a callback.  When set, SYNCHRONOUS indicates
+  //   that there may be an associated trace context, otherwise 
+  //   data points are considered asynchronous.  Asynchronous Counts
```

Comment:
What `Count` does this redder to?

Reply:
I clarified that this refers to `DataPoint.Histogram.Count` or `DataPoint.Summary.Count`.